katcp/resource.py

```diff
@@ -595,10 +595,15 @@ def parent_name(self):
 
     @property
     def name(self):
-        """Name of this KATCP resource"""
+        """Name of this KATCPSensor"""
         return self._name
 
     @property
+    def python_id(self):
```

Comment:
I am happy with the code.
Personally I do not like the method name python_id maybe normalised_name
@theunsa and I chatted we have a hand full of possible names.
 We think @lvdheever can decide on a nice name.